cypress/integration/instance/InstanceEdit.ts

```diff
@@ -50,5 +50,67 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
             });
         });
     });
+
+    it.only("Switch from 'Webhook' to 'Ignore' Error handler", () => {
```

Comment:
Is this `it.only` intentional?

Reply:
no, it is a mistake. I will remove it, thank you